review/templates/review/_watermarktable.html

```diff
@@ -0,0 +1,62 @@
+{% load review %}
+<head>
+  <link rel="stylesheet" href="table.css">
+</head>
```

Comment:
The `<head>` element should not be used in the HTML body. Django template blocks would be the right way to add contents to the head. Though, at the moment this template is included into another template and it does not inherit anything (which is required for blocks).